### PR TITLE
add timestamp to all calendar cells

### PR DESCRIPTION
**Pull request by fritzmg**
Whenever I have to use the calendar module for a website, the layout/design concept usually requires additional information to be shown in the calendar, e.g. days of previous and following months or week numbers for example. 

However, currently this is pretty tedious to implement. You either have to use ugly code within the calendar template or duplicate some core module code for your own calendar module.

It would be a lot easier, if each cell already had a timestamp, with which you can simply do anything you want.

### CODE REVIEW

File: calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php [hunk 1]
```diff
@@ -239,6 +239,9 @@ protected function compileWeeks()
 			$strClass = ($intCurrentDay < 2) ? ' weekend' : '';
 			$strClass .= ($i == 1 || $i == 8 || $i == 15 || $i == 22 || $i == 29 || $i == 36) ? ' col_first' : '';
 			$strClass .= ($i == 7 || $i == 14 || $i == 21 || $i == 28 || $i == 35 || $i == 42) ? ' col_last' : '';
+			
+			// Add timestamp to all cells
+			$arrDays[$strWeekClass][$i]['timestamp'] = strtotime(($intDay - 1) . ' day', $this->Date->monthBegin);
```
Comment (leofeyer) [inline calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php:244]: Does this expression compile to e.g. `6 day`?

Comment (fritzmg) [reply calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php:244]: Yes, or `-4 day` etc.

Comment (leofeyer) [reply calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php:244]: I know that `strtotime()` is very lax with its textual dates, but I didn't know that `6 day` is treated as `+6 days`. I assume you are sure about this? 😄

Comment (fritzmg) [reply calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php:244]: Well I am already using it this way at least ;). With a custom module where I simply copy & pasted the compileWeeks() function and included the above change.

**Comment by leofeyer** [comment]
What about adding a standardized date instead of the Unix timestamp?

```php
$arrDays[$strWeekClass][$i]['date'] = date('Y-m-d', $this->Date->tstamp);
```

This could be transformed into a Unix timestamp using `strtotime()` in case someone explicitly needs a timestamp.

**Comment by fritzmg** [comment]
Sure, why not, does not really matter in the end.

**Comment by leofeyer** [comment]
Thank you @fritzmg.

**Pull request merged**